src/Core.Scripts/src/ExportedMethods.ts

```diff
@@ -2,6 +2,7 @@ import { Microsoft as LoggerFile } from './Utilities/Logger';
 import { Microsoft as AttributesFile } from './Utilities/Attributes';
 import { Microsoft as FluentDialogFile } from './Components/Dialog/FluentDialog';
 import { Microsoft as FluentTabsFile } from './Components/Tabs/FluentTabs';
+import { Microsoft as FluentMultiSplitterFile } from './Components/Splitter/FluentMultiSplitter';
```

Comment:
Why do we use the `File`-suffix here? Seems off. Maybe use `Script`-suffix instead?

Reply:
This is only internal to this file. And I need to use a suffix to distinct the "file" and the "import". We can change this suffix in another PR.